lib/rucio/core/authentication.py

```diff
@@ -59,12 +62,10 @@ def generate_key(token, session=None):
     return generate_key
 
 
-TOKENREGION = make_region(
-    function_key_generator=token_key_generator
-).configure(
-    'dogpile.cache.memory',
-    expiration_time=3600
-)
+if config_get_bool('cache', 'use_external_cache_for_auth_tokens', default=False):
```

Comment:
Just one comment, please add this also to the documentation
http://rucio.cern.ch/documentation/configuration_parameters